app/csv/Charge.scala

```diff
@@ -68,6 +68,25 @@ Charge_day // day we're running for
      and RatePlanCharge.Name = 'Holiday Credit')
     */
 
+  val Rate_Plan_Charge__Charged_Through_Date = 0
```

Comment:
did you mean to put these back in? maybe you are using them for the data lake import?